assets/queries/cloudFormation/filter_and_alarm_exist_for_aws_management_console_authentication_failures/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "Filter_And_Alarm_Exist_For_AWS_Management_Console_Authentication_Failures",
+  "queryName": "Filter And Alarm Not Exists For AWS Management Console Authentication Failures",
+  "severity": "MEDIUM",
+  "category": "Monitoring",
+  "descriptionText": "A log metric filter and alarm should exist for AWS Management Console Authentication Failures",
+  "descriptionUrl": "https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cis-controls.html"
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline